www/front_src/src/Header/PollerMenu/index.tsx

```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles((theme) => ({
     borderWidth: '1px',
     color: theme.palette.common.white,
     display: 'flex',
-    gap: theme.spacing(1),
+    gap: theme.spacing(9),
```

Comment:
Does it work with a thousand poller issues ?

Reply:
With justifyContent it's better , thank's Tom ;)